docs/development/decision-records/2023-02-09-release-process/README.md

```diff
@@ -0,0 +1,117 @@
+# Release process of tractusx-edc
+
+## Decision
+
+To improve stability, reproducibility and maintainability of releases, tractusx-edc will undergo the following changes:
+
+- use EDC `SNAPSHOT` builds during development
+- use release versions of EDC in releases. Release branches must not change upstream dependency versions, unless there
+  is a clear and concise reason to do so.
+- slightly update branching model
+- if possible, bugs/defects should be fixed on `develop` and be backported to the respective `hotfix/` branch
+- Catena-X/Tractus-X will only provide hotfixes for critical security bugs as defined by the committers for the
+  currently released version. Nothing else.
+- feature development happens _in developers' forks only_ to keep the Git reflog of the `origin` clean.
+
+## Rationale
+
+Having releases depend on snapshot versions of upstream projects, such as EDC, is inherently dangerous, particularly
+when that dependency has not yet reached a final state and breaking changes are to be expected. Most problems will stem
+from breaking changes, such as Java SPIs, APIs and changes in service contracts.
+
+Up until now, the only way out was cherry-picking, which is extremely cumbersome and error-prone, and requires a
+parallel build pipeline to publish the cherry-picked artifacts of EDC (and potentially others). With the approach
+presented here, cherry-picking is still an option, but there are easier alternatives to it. 
+
+Every release version published by tractusx-edc must be reproducible at any time.
+
+## Approach
+
+### Use EDC `SNAPSHOT` versions during development
+
+During feature development we only use `-SNAPSHOT` versions of EDC packages. It is assumed that when the build breaks
+due to changes in upstream, the fix can be done quickly and easily, much more so than working off technical
+debt that would otherwise accumulate over several months. Builds on `develop` are therefore _not repeatable_, but that
```

Comment:
the problem with Snapshots is that they could get overwritten or deleted. 
That means, that EDC's `0.0.1-SNAPSHOT` versions will get overwritten on each EDC build. Also, when OSSRH decides to clean out their storage, and one of our builds depends on a now-deleted "named" snapshot (e.g. `0.0.1-20230315-SNAPSHOT`), we're screwed.

However, during feature development, that is not an issue, because `develop` will depend on EDC `X.Y.Z-SNAPSHOT`, which is always there, but it may be different from day to day.

Reply:
Thanks for the explanation! That makes things more clear for me!